client/dashboard/components/page-layout/index.tsx

```diff
@@ -50,4 +53,12 @@ function PageLayout( {
 	);
 }
 
+function CallOutParent( { children }: { children: React.ReactNode } ) {
```

Comment:
I'm not using a modal to implement the "sneeze guard". It looks like a modal, but it's a bit different. For example pressing escape doesn't navigate you away from the page.

This sort of call out will need special focus management though, to ensure the user doesn't tab into the content behind the "sneeze guard". I'll see if there's anything I can commandeer from core for this.